jdl/validators/jdl-without-application-validator.js

```diff
@@ -202,7 +206,7 @@ function checkForAbsentEntities({ jdlRelationship, doesEntityExist, skippedUserM
   }
 }
 function isUserManagementEntity(entityName) {
-  return entityName.toLowerCase() === 'user' || entityName.toLowerCase() === 'authority';
+  return builtInEntities.has(entityName.toLowerCase());
```

Comment:
For now this one works, but semantically speaking this change is a little bit risky: WDYT?

Reply:
I could undo this. I look up this approach from the Jhipster codebase. I don't if this is better or worse..